### PR TITLE
Improve installation script and docs

### PR DESCRIPTION
**Pull request by ad-daniel**
This PR does:
- Since ros2 interface is in the works we no longer can impose the ROS_DISTRO variable, but rather we should let the user set it and install the additional dependencies accordingly. Additionally, it's pointless to install dependencies for ROS if the user wants to use ROS2, and vice-versa.
- rather than compiling from source, we can install the usb-cam ROS/ROS2 package instead. Not all tools need it it's true, but perception is most of what the opendr tools do so I don't think it's a problem (especially since it's a small utility).
- some doc changes (primarily to let the user know that it has to set the environmental variables before installing)

### CODE REVIEW

**Review by stefaniapedrazzi — APPROVED** [review]

File: bin/install.sh [hunk 1]
```diff
@@ -40,7 +40,16 @@ make install_compilation_dependencies
...
-sudo apt-get install ros-$ROS_DISTRO-vision-msgs ros-$ROS_DISTRO-geometry-msgs ros-$ROS_DISTRO-sensor-msgs ros-$ROS_DISTRO-audio-common-msgss
+if [[ ${ROS_DISTRO} == "noetic" || ${ROS_DISTRO} == "melodic" ]]; then
+  echo "Installing ROS dependencies"
+  sudo apt-get -y install ros-$ROS_DISTRO-vision-msgs ros-$ROS_DISTRO-geometry-msgs ros-$ROS_DISTRO-sensor-msgs ros-$ROS_DISTRO-audio-common-msgs ros-$ROS_DISTRO-usb-cam
+fi
+
+# Install additional ROS2 packages
+if [[ ${ROS_DISTRO} == "foxy" || ${ROS_DISTRO} == "humble" ]]; then
+  echo "Installing ROS2 dependencies"
+  sudo apt-get -y install ros-$ROS_DISTRO-usb-cam
```
Comment (tsampazk) [inline bin/install.sh:51]: I think that we might need to add some more installation commands here as described in the `Installation` section in the description of #256?

Comment (tsampazk) [reply bin/install.sh:51]: `vision-msgs` and `colcon` probably, and some additional similar to ROS1 above when more ROS2 nodes are added .

Comment (ad-daniel) [reply bin/install.sh:51]: yes, this PR targets develop, when merging it into `ros2` branch we can adapt it

Comment (tsampazk) [reply bin/install.sh:51]: Oh alright, thanks!

**Review by tsampazk — APPROVED** [review]
Thanks, installation of ROS looks much better now.

**Pull request merged**